go.mod

```diff
@@ -159,7 +159,7 @@ require (
 	github.com/jackc/pgx/v5 v5.3.1
 	github.com/jaegertracing/jaeger v1.18.1
 	github.com/jordan-wright/email v4.0.1-0.20210109023952-943e75fe5223+incompatible
-	github.com/jordanlewis/gcassert v0.0.0-20221027203946-81f097ad35a0
+	github.com/jordanlewis/gcassert v0.0.0-20230607150025-9012d8e73bdd
```

Comment:
We don't need to/shouldn't be updating `gcassert` any more, right? The `bazel` patches that were applied are all faulty and we don't seem to be using them anyway.

Reply:
We still need to update gcassert to pull changes where we get the logs and can run in debug mode. In the current version we do not capture build logs.